CefSharp.Core/Internals/CefTaskScheduler.h

```diff
@@ -42,7 +43,11 @@ namespace CefSharp
                 }
                 catch (Exception^)
                 {
-
+                    // This should never ever happen.
```

Comment:
Ahh, true. There really is not nice place to raise a `.Net` error from within `CEF`. Have to think about that.